assets/css/style.css

```diff
@@ -185,7 +185,7 @@ body {
 
 /* === 3.1 HOME BODY === */
 
-body.home {
+/*body.home {
```

Comment:
If you think that we'll need this later, can you add comments as to why you commented this out? OR if you don't think we'll need it, just nuke it :)